kms/pkcs11/pkcs11_test.go

```diff
@@ -491,6 +492,86 @@ func TestPKCS11_CreateSigner(t *testing.T) {
 	}
 }
 
+func TestPKCS11_CreateDecrypter(t *testing.T) {
+	k := setupPKCS11(t)
+	data := []byte("buggy-coheir-RUBRIC-rabbet-liberal-eaglet-khartoum-stagger")
+
+	type args struct {
+		req *apiv1.CreateDecrypterRequest
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{"RSA", args{&apiv1.CreateDecrypterRequest{
+			DecryptionKey: "pkcs11:id=7371;object=rsa-key",
+		}}, false},
+		{"RSA PSS", args{&apiv1.CreateDecrypterRequest{
+			DecryptionKey: "pkcs11:id=7372;object=rsa-pss-key",
+		}}, false},
```

Comment:
I think the test name should be `RSA OAEP`? For the key object it's probably OK to use the `rss-pss-key` object and not having to add a new one (for now).

Reply:
Each test tests both pkcs#1 v1.5 and RSA OAEP, this is basically using a regular RSA key. 

On PKCS#11 there's no difference on the keys, the difference happens at signing time. But for other KMSs you create a key for RSA PSS and it always signs using PSS. This is why the API makes distinctions.

Reply:
Ah, got it. The PSS threw me off. Looks good then!